cfe_internal/enterprise/main.cf

```diff
@@ -32,7 +32,9 @@ bundle agent cfe_internal_enterprise_main
       handle => "cfe_internal_management_php_runalerts",
       comment => "To run PHP runalerts to check bundle status on SQL and Sketch.
                  ENT-5432: must run after cfe_internal_enterprise_maintenance bundle
-                 so that active_hub class is determined";
```

Comment:
I have FUD with multi-line comment strings and reporting, so I usually adjust them with `concat()`.